src/components/submission/list.vue

```diff
@@ -269,15 +275,19 @@ export default {
   methods: {
     fetchChunk(skip, clear) {
       this.refreshing = skip === 0 && !clear;
-      const top = this.top(skip);
-      const query = { $top: top, $skip: skip, $count: true, $wkt: true };
-      if (this.odataFilter != null) query.$filter = this.odataFilter;
       this.odata.request({
         url: apiPaths.odataSubmissions(
           this.projectId,
           this.xmlFormId,
           this.draft,
-          query
+          {
+            $top: this.top(skip),
+            $skip: skip,
+            $count: true,
+            $wkt: true,
+            $filter: this.odataFilter,
```

Comment:
Previously, `$filter` was not specified at all if `this.odataFilter == null`. However, there's no harm in always specifying it. If the value of a query parameter is `null` or `undefined`, then `apiPaths.odataSubmission()` won't include it in the query string. I think the code reads more easily with all the query parameters grouped together like this.